Gemfile

```diff
@@ -57,7 +57,7 @@ gem 'rake'
 gem 'retries'
 gem 'ruby-prof'
 gem 'rubyzip'
-gem 'zip_tricks', '~> 5.3'
+gem 'zip_tricks', '5.3.1'
```

Comment:
Can you add a comment here about the pin?

Reply:
Done.